setup.py

```diff
@@ -24,7 +24,7 @@
     # Needed to actually package something
     packages=find_packages(exclude=["test", "test.*"]),
     # Needed for dependencies
-    install_requires=["python-can>=4.0.0"],
+    install_requires=["python-can>=4.0.0", "pytest", "pytest-mock"],
```

Comment:
These shouldn't be in the installation requirements but more in `tests_require` and/or `extras_require={"test": [...]}`

Reply:
fixed